frontend/src/metabase/search/components/DropdownSidebarFilter/DropdownSidebarFilter.tsx

```diff
@@ -119,10 +119,16 @@ export const DropdownSidebarFilter = ({
           <Button
             data-testid="sidebar-filter-dropdown-button"
             compact
+            mr="0.25rem"
+            size="xs"
             c="inherit"
             variant="subtle"
             onClick={onClearFilter}
-            leftIcon={<Icon name={getDropdownIcon()} />}
+            leftIcon={
+              <Center m="-0.25rem">
+                <Icon size={16} name={getDropdownIcon()} />
```

Comment:
The icon shrinks when the label is long (it needs `flex: 0 0 auto`).
(this is an actual Polish translation of "Last edited by")

![image](https://github.com/metabase/metabase/assets/6830683/010a6f2c-d3b0-4879-837d-60235f57fd3e)

Reply:
Solved, removed an overflow that was causing the issue